server/server.go

```diff
@@ -129,6 +129,8 @@ func portInUse(hostPort string) bool {
 }
 
 func newServerFromHandler(cfg Config, e *sqle.Engine, sm *SessionManager, handler mysql.Handler) (*Server, error) {
+	handler = withChain(handler)
```

Comment:
This should be a configuration option on `cfg`, not hardcoded here. We don't want the interceptor chain installed by default (even with an empty chain)

Reply:
Done